Mathlib/Data/Sym/Basic.lean

```diff
@@ -534,6 +534,19 @@ theorem mem_append_iff {s' : Sym α m} : a ∈ s.append s' ↔ a ∈ s ∨ a ∈
   Multiset.mem_add
 #align sym.mem_append_iff Sym.mem_append_iff
 
+/-- Defines an equivalence between `α` and `Sym α 1`. -/
```

Comment:
This is devoid of information. I already know this from the *type* of the declaration. However I would like to know what's going on inside the *body*. Can you change it to something like 
```suggestion
/-- `a ↦ {a}` as an equivalence between `α` and `Sym α 1`. -/
```